oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/components/TimelineCommentEvent.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { memo } from "react";
+import { i18next } from "@translations/oarepo_requests_ui/i18next";
+import { Feed, Image } from "semantic-ui-react";
+import _has from "lodash/has";
+import sanitizeHtml from "sanitize-html";
+import PropTypes from "prop-types";
+import { toRelativeTime } from "react-invenio-forms";
+
+const TimelineCommentEvent = ({ event }) => {
+  const createdBy = event?.expanded?.created_by;
+  const creatorLabel = createdBy?.username || createdBy?.email;
+  return (
+    <div className="requests comment-event-container">
+      <Feed.Event key={event.id}>
+        {createdBy?.links?.avatar && (
+          <div className="requests comment-event-avatar">
+            <Image
+              src={createdBy?.links?.avatar}
+              alt={i18next.t("User avatar")}
+            />
+          </div>
+        )}
+        <Feed.Content>
+          <Feed.Summary>
+            <b>{creatorLabel}</b>
+            <Feed.Date>
+              {i18next.t("requestCommented")}{" "}
+              {toRelativeTime(event.created, i18next.language)}
+            </Feed.Date>
+          </Feed.Summary>
+          {_has(event.payload, "content") && (
+            <Feed.Extra text>
+              <div
+                dangerouslySetInnerHTML={{
+                  __html: sanitizeHtml(event.payload.content),
```

Comment:
Use req form cfg api to fetch sanitize config (allowed tags)

Reply:
@mirekys This is now a bit problematic, as this component has no knowledge of any given request and the requests api end point calls to for example /requests/configs/publish_draft

Here are the available endpoints in terms of configs

communities_dashboard.form_config                    GET        /configs/me/communities/form
documents.form_config                                GET        /configs/docs/form
global_search_ui.form_config                         GET        /configs/search/form
oai-batch-ui.form_config                             GET        /configs/oai/harvest/batches/form
oai-harvester-ui.form_config                         GET        /configs/oai/harvest/harvesters/form
oai-record-ui.form_config                            GET        /configs/oai/harvest/records/form
oai-run-ui.form_config                               GET        /configs/oai/harvest/runs/form
oarepo_communities.form_config                       GET        /configs/communities/form
oarepo_requests_form_config.form_config              GET        /requests/configs/<request_type>
oarepo_vocabularies_ui.form_config                   GET        /configs/vocabularies/<vocabulary_type>/form
records_dashboard.form_config                        GET        /configs/me/records/form
requests_dashboard.form_config                       GET        /configs/me/requests/form

In my opinion, there isn't appropriate end point to call to at this point. There should probably exist a separate end point that provides just this sanitization data, that will be a fixed URL that will reside in oarepo ui or some other library you can be sure exists. 

I could of course just call to /requests/configs/publish_draft as I am guessing this will always be available. Do you think that this is a valid solution please?